ddtrace/internal/compat.py

```diff
@@ -86,26 +87,17 @@
 else:
     pattern_type = re._pattern_type  # type: ignore[misc,attr-defined]
 
-try:
-    from inspect import getargspec as getfullargspec
-
-    def is_not_void_function(f, argspec):
-        return argspec.args or argspec.varargs or argspec.keywords or argspec.defaults or isgeneratorfunction(f)
-
 
```

Comment:
Can we keep `inspect.getargspec()` for Python 2?
```suggestion
if PY2:
    from inspect import getargspec as getfullargspec
    def is_not_void_function(f, argspec):
        return argspec.args or argspec.varargs or argspec.keywords or argspec.defaults or isgeneratorfunction(f)
else:
    from inspect import getfullargspec  # type: ignore[assignment]  # noqa: F401

    def is_not_void_function(f, argspec):
        return (
            argspec.args
            or argspec.varargs
            or argspec.varkw
            or argspec.defaults
            or argspec.kwonlyargs
            or argspec.kwonlydefaults
            or isgeneratorfunction(f)
        )
```

Reply:
NVM - your change is fine as is.